src/index.html

```diff
@@ -3,11 +3,282 @@
+                  alt="Range & Integrated lights"
+                  class="gallery__image gallery__image--big"
+                >
+              </div>
+
+              <div class="gallery__image-wrapper" data-aos="flip-left">
+                <img
+                  src="./images/image 7.png"
+                  alt="Range & Integrated lights"
+                  class="gallery__image"
+                >
+              </div>
+            </div>
+            <h3 class="gallery__title" data-aos="fade-up">Range & Integrated lights</h3>
+            <p class="gallery__desc" data-aos="fade-up">The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.</p>
+          </div>
+
+          <div class="gallery details__item">
+            <div class="gallery__images gallery__images--reverse">
+              <div class="gallery__image-wrapper" data-aos="flip-left">
+                <img
+                  src="./images/image 11.png"
+                  alt="Hydraulic disc brakes & Lightweight"
+                  class="gallery__image"
+                >
+              </div>
+
+              <div class="gallery__image-wrapper" data-aos="flip-left">
+                <img
+                  src="./images/image 10.png"
+                  alt="Hydraulic disc brakes & Lightweight"
+                  class="gallery__image  gallery__image--big"
+                >
+              </div>
+            </div>
+            <h3 class="gallery__title" data-aos="fade-up">Hydraulic disc brakes & Lightweight</h3>
+            <p class="gallery__desc" data-aos="fade-up">Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.</p>
+          </div>
+        </div>
+        <a
+          href="https://www.mybike.pl/rowery"
+          target="_blank"
+          class="page__btn details__link"
+          data-aos="fade-up"
+        >
+          Explore
+        </a>
+      </section>
+    </main>
+    <footer class="contact" id="contacts">
+      <div class="contact__container">
+        <h2 class="contact__title page__h2" data-aos="fade-up">Contact us</h2>
+        <form action="#" class="contact__form" data-aos="fade-right">
+          <!-- linthtml-disable -->
+          <input
+            type="text"
+            name="name"
+            id="name"
+            placeholder="Name"
+            class="contact__input"
+            required
+          >
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="Email"
+            class="contact__input"
+            required
+          >
+          <textarea
+            name="msg"
+            id="msg"
+            placeholder="Message"
+            class="contact__textarea"
```

Comment:
I will add `resize: none` rule to the textarea, because right now I could resize it in any size.

Reply:
there is resize: vertical; in contact.scss

Reply:
is vertical shouldn't be used ? should i change it to none?

Reply:
Yeap, because you could do such things
<img width="1511" alt="image" src="https://user-images.githubusercontent.com/32794096/225576388-e3654115-b8ef-45f6-9f1a-1fe8928e5d20.png">